docs/upgrade.html

```diff
@@ -430,6 +430,9 @@ <h5><a id="upgrade_servers_400_notable" href="#upgrade_servers_400_notable">Nota
                     <li> See <a href="https://cwiki.apache.org/confluence/x/B40ODg">KIP-890</a> and
                     <a href="https://cwiki.apache.org/confluence/x/8ItyEg">KIP-1050</a> for more details </li>
                 </ul>
+                <li>
+                    The <code>window.size.ms</code> and <code>window.inner.serde.class</code> in stream config are deprecated.
```

Comment:
```suggestion
                    The <code>window.size.ms</code> and <code>window.inner.serde.class</code> in `StreamsConfig` are deprecated. Use the corresponding string constants defined in `TimeWindowedSerializer`, `TimeWindowedDeserializer`, `SessionWindowedSerializer` and `SessionWindowedDeserializer` instead.
```

Something like that? Could also be the actual string constants. But it's good to make these notes actionable.